src/rajasekhar/findFinalDirection/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './findFinalDirection';
```

Comment:
instead using default you must give this a meaningful name

Reply:
I want this file to pass what I got from my `findFinalDirection` file. I have only one thing to import and one thing to export, and no requirement to pass many sub items from my component. So I have passed the default as default. 
Here the implementation is 
`export { default } from './findFinalDirection';`
so my import function can be
```import findFinalDirection from './index';```
If I choose other way like
`export { default  as findFinalDirection} from './findFinalDirection';`
my import should be written like
```import { findFinalDirection } from './index';```
Since I am exporting one function and to avoid one more curly brace. I choose to export as default the default I have imported.

Reply:
if you want to pass everything from `findFinalDirection` you should use export * from './findFinalDirection';
if there is only one function still you can use above suggestion but using `default` for export doesn't fulfill naming  
you should use * or any meaningful name other than `default` .

Reply:
If I use `export * from './findFinalDirection';`, it will not set the default coming from the module so that won't help.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
This is the equivalent for what I am doing (`export { default } from './findFinalDirection';`).
```
import findFinalDirection from './findFinalDirection';
export default findFinalDirection;
```

Reply:
@zulqer-nain any points to add here?